spyderlib/config.py

```diff
@@ -169,6 +169,7 @@ def is_ubuntu():
               'animated_docks': True,
               'prompt_on_exit': False,
               'panes_locked': True,
+              'interface_language': load_lang_conf(),

```

Comment:
Please remove this option from here to finally merge this PR :-)

If I understand the code correctly:

1. It'll be saved to `spyder.ini` by the `save_lang` method of `MainConfigPage`
2. It can't be retrieved by `CONF` directly.

So there's no need to have it here.